app/blog/2018-03-14-polymer-3-0.md

```diff
@@ -0,0 +1,330 @@
+---
+title:  "Hands-on with the latest Polymer 3.0 preview"
+---
+
+In a previous [blog post on paths and names](2018-02-26-3.0-preview-paths-and-names), we announced some changes to the way we handle module imports in Polymer 3.0, and described why we made those changes. We're happy to announce that you can try out the new import syntax in the latest Polymer 3.0 preview!
+
+* [Simpler module imports](#packagenames)
+* [Renamed base element export in polymer-element.js](#renamed)
```

Comment:
Maybe add a new section (or expand/rename this section?). 

There's one other change they're doing at the same time, which is renaming "polymer.js" to "polymer-legacy.js". 

Probably most people aren't using this directly, but like renaming Element => PolymerElement, it's a breaking change for anyone who's using it.